.github/workflows/pr.yml

```diff
@@ -9,14 +11,14 @@ on:
     tags:
       - sdk/v*.*.*
     paths-ignore:
+      # Skip CI checks if the only things that have changed are CHANGELOG entries (which is typically the case when a PR
+      # or Push is occurring as part of a release process).
       - 'CHANGELOG.md'
       - 'CHANGELOG_PENDING.md'
   repository_dispatch:
     types:
       - run-acceptance-tests-command
 
-# TODO[pulumi/java#10] protect jobs from crypto mining PR attacks
-# before making the repo public.
```

Comment:
This issue is long closed so this comment is not relevant; I took the opportunity to clean it up as part of this change -- hope this is OK!

Reply:
Hah, I stumbled over thsi so many times, and always forget to do something about it.  Thanks for removing it.  We're definitely protected from this by existing CI infrastructure already.